.vscode/settings.json

```diff
@@ -24,4 +24,5 @@
   "javascript.preferences.importModuleSpecifier": "relative",
   "typescript.preferences.importModuleSpecifier": "relative",
   "prettier.configPath": ".prettierrc.js",
+  "editor.formatOnSave": true,
```

Comment:
This should be removed because it is a file that has nothing to do with this PR.

Reply:
I will fix it so don't care about it.